Source/WebCore/html/HTMLLIElement.cpp

```diff
@@ -89,19 +89,13 @@ void HTMLLIElement::collectPresentationalHintsForAttribute(const QualifiedName&
             else if (valueLowerCase == "none"_s)
                 addPropertyToPresentationalHintStyle(style, CSSPropertyListStyleType, CSSValueNone);
         }
+    } else if (name == valueAttr) {
+        if (auto parsedValue = parseHTMLInteger(value))
+            addPropertyToPresentationalHintStyle(style, CSSPropertyCounterSet, makeString("list-item "_s, *parsedValue));
```

Comment:
There's an extra space in the string here. That would cause `value=""` not to work at all, right?

If this is not causing test failures, please add a test that would've failed.

Reply:
You want it to be `counter-set: list-item 4` not `counter-set: list-item4`. `imported/w3c/web-platform-tests/css/css-lists/li-value-counter-reset-001.html` would have failed otherwise.

`value=""` would just fallback to normal rendering as `parseHTMLInteger` would give a parse error. See: https://html.spec.whatwg.org/#the-li-element

Reply:
Of course, you are right.